syntax = "proto3";

package test.common.access_log;

import "test/fuzz/common.proto";

// Structured input for access_log_formatter_fuzz_test.

message TestCase {
  string format = 1;
  test.fuzz.Headers request_headers = 2;
  test.fuzz.Headers response_headers = 3;
  test.fuzz.Headers response_trailers = 4;
  test.fuzz.RequestInfo request_info = 5;
}
